styles/main.css

```diff
@@ -99,6 +95,17 @@ li {
   background-color: #008552;
 }
 
+@media screen and (min-width: 992px) {
+  .title,
+  .description {
+    margin-inline: 22rem;
+  }
+
+  .links {
+    width: 20vw;
+  }
+}
+
 /* End General Rules */
 
 /* Start Header Section */
```

Comment:
- [ ] Please add a `background-color` to the `header` element. right now it's transparent and causes things behind the header to be visible under the text of the header which is visually distracting. 
- [ ] Or you could make sure that the header stays at the top of the page and doesn't stay fixed on the page.
## Issue (transparent background)
![image](https://user-images.githubusercontent.com/18730106/144493707-dc8bb72b-f312-4527-8b00-9ed5bc2af0e9.png)